data/phi0690/phi003/phi0690.phi003.perseus-eng2.xml

```diff
@@ -38,17 +37,18 @@
 	           <biblStruct>
 	              <monogr>
 	                 <author>Vergil</author>
-	                 <title>Aeneid</title>
+	                 <title>The Aeneid of Virgil</title>
 	                 <respStmt>
 	                    <name>Theodore C. Williams</name>
-	                    <resp>trans.</resp>
+	                    <resp>translator</resp>
```

Comment:
Probably preferable practice to use `<editor role="translator">` given what Eldarion is doing with `<respStmt>` tags